build/Common.props

```diff
@@ -8,6 +8,8 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```

Comment:
You probably want to rename this. `TargetFrameworks` is a well-known thing in SDK projects we shouldn't co-opt. Maybe something like `RepoStandardTargetFrameworks`?